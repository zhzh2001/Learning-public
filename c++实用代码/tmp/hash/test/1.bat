C:\Dev-Cpp\bin\g++ hash.cpp -o hash -O2
C:\Dev-Cpp\bin\g++ hash_stl.cpp -o hash_stl -O2
C:\Dev-Cpp\bin\g++ hash_stl(map).cpp -o hash_stl(map) -O2
pause
