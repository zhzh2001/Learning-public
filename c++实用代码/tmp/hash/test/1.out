10
time=4482
it=895
sum=-671657525
